packages/components/src/components/editor/browser/browser.stories.tsx

```diff
@@ -82,36 +81,39 @@ const DefaultBrowser = ({ applyFn, applyBtn, initSearch }: DefaultBrowserProps)
   );
 };
 
-const QuickActionPopover = () => {
+const QuickActionPopover = ({ additionalInfo }: { additionalInfo?: React.ReactNode }) => {
```

Comment:
As we don't know if we really do it this way, I would not call this `QuickActionPopover`.
And this here is an example what you can do.

I would just call it `CmsHeader`. 
And as it is just an example code you don't have to put the info text as property. Just write it in the component itself.